add link to github-repo on the websites

niccokunzmann opened the pull request:
fix #54

Satz wird eingefügt: add link to github-repo on the websites

niccokunzmann committed:
fix #54 - add link to github-repo on the websites

SvenRoederer commented:
@niccokunzmann kannst du die Commit-message etwas aussagekräftiger machen?
Dann man nicht erst bei Github nachsehen muss, was mit "#54" gemeint ist.

niccokunzmann commented:
@SvenRoederer ich habe den Titel eingefügt

SvenRoederer commented on ca/templates/_layout.html:
```diff
@@ -30,6 +30,6 @@ <h2>
       </div>
     </div>
-
+    ca.berlin.freifunk.net wird von der Berliner Freifunk-Gemeinschaft <a href="https://github.com/freifunk-berlin/ca.berlin.freifunk.net">auf GitHub</a> entwickelt. Du kannst mithelfen und die Software verbessern.
```
to hard-code "ca.berlin.freifunk.net" ist eher unpraktisch. Denn der Code wird auch bei "tunnel.berlin.freifunk.net" genutzt.

P.S: ich dachte, ich schrieb das schon mal ....

SvenRoederer replied:
Vorschlag: " ca.berlin.freifunk.net" ändern in "dieser Web-service"

niccokunzmann committed:
Remove domain name

https://github.com/freifunk-berlin/ca.berlin.freifunk.net/pull/58/files#r175909085

niccokunzmann commented:
@SvenRoederer Ich habe es in 8bb5b5f adressiert.

bobster-galore commented:
Would be nice! How to progress?

SvenRoederer commented:
jetzt wo ich den Kommentar von @niccokunzmann wieder sehe, fällt mir wieder ein, was hier Schuld war: die Arbeit an Hedy 1.0.0 ....

niccokunzmann commented:
Danke!
